.github/actions/run-ios-simulator/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+      "target": "ES2018",
```

Comment:
This was changed to accomodate the interface of `RegExpExecArray` that contains `groups`